### PR TITLE
Windows path correction

### PR DESCRIPTION
**Pull request by niteskum**
Windows network path also need to be converted to windows style path
@jha-g  @narayani28  please review

### CODE REVIEW

File: src/extensions/default/OpenWithExternalApplication/main.js [hunk 1]
```diff
@@ -49,7 +49,8 @@ define(function (require, exports, module) {
     var extensionToExtApplicationMap = {};
 
     function convertUnixPathToWindowsPath(path) {
-        if (brackets.platform === "win" && path && path[1] === ":" && path[2] === "/") {
+        if (brackets.platform === "win"
```
Comment (jha-g) [inline src/extensions/default/OpenWithExternalApplication/main.js]: What is the reason for this change? Windows network paths?

Comment (niteskum) [reply src/extensions/default/OpenWithExternalApplication/main.js]: when we open network file with some external application, it does not accept path with forward-slash "/", so need to converted to windows style.

e.g

Illustrator //<network_path_with_forward_slash>/example.ai    this does not work

so have to give like : Illustrator \\<network_path_with_backward_slash>\example.ai

Comment (jha-g) [reply src/extensions/default/OpenWithExternalApplication/main.js]: Can't we use the condition  `brackets.platform === "win" && FileSystem.isAbsolutePath()`.

Comment (jha-g) [reply src/extensions/default/OpenWithExternalApplication/main.js]: Also are we planning to remove leading slashes from paths like `/C:/notepad/notepad.exe`?
And more than two leading slashes do not work for network path, so we are covered there.

Comment (niteskum) [reply src/extensions/default/OpenWithExternalApplication/main.js]: Here we will never have local path with leading slashes so not required.

**Review by g-217 — APPROVED** [review]
